common-treble.mk

```diff
@@ -12,8 +12,6 @@ PRODUCT_PACKAGES += \
     vendor.qti.hardware.display.mapper@2.0.vendor
 
 # Graphics allocator/mapper
-ifeq ($(TARGET_HARDWARE_GRAPHICS_V3),true)
```

Comment:
Is this true for Q too? Are Loire and Tone still somewhat alive/buildable here?

Reply:
I am checking these commits agian, I think we do not need all of them.